src/data/faq_de.json

```diff
@@ -890,27 +890,27 @@
                             {
                                 "title": "Wie verwalte ich Auffrischimpfungen in der CWA?",
                                 "anchor": "vac_cert_booster_cwa",
                                 "active": false,
                                 "textblock": [
-                                    "Seit Version 2.9 unterstützt die Corona-Warn-App auch den Import von Zertifikaten, die eine Auffrischimpfung dokumentieren. (siehe Blog Artikel: <a href='../../blog/2021-09-08-cwa-version-2-9/' target='_blank' >Corona-Warn-App Version 2.9: Veranstalter*innen können Gäste im Auftrag des Gesundheitsamtes über die Check-in-Funktion warnen</a>)",
-                                    "Seit Version 2.10 erhalten Anwender einen Hinweis, ggf. eine Auffrischimpfung in Betracht zu ziehen. Der Hinweis basiert auf nationalen Regeln, die in Absprache mit dem Robert Koch-Institut (RKI) implementiert wurden. Diese Regeln werden fortlaufend geprüft und entsprechend den Entwicklungen und neuen Erkenntnissen angepasst. (siehe Blog Artikel: <a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank' >Mit Version 2.10 ist die Corona-Warn-App bereit, schnell über Auffrischimpfungen zu informieren.</a>)",
-                                    "Bitte beachten Sie: Das Zertifikat zur Auffrischimpfung ist nur gültig, wenn Sie auch die nötige Grundimmunisierung durch das entsprechende letzte Zertifikat der Impfreihe nachweisen können. Löschen Sie bitte die anderen Impfzertifikate nicht aus der CWA! Da bisher noch keine einheitlichen EU-Standards zu diesem Thema festgelegt wurden, sollte man insbesondere bei Reisen ins Ausland auch die Zertifikate der Grundimmunisierung mit sich führen, sei es in der CWA, als PDF, oder als Papierausdruck. Wir empfehlen daher alle Impfzertifikate in der App zu behalten, auch wenn diese nicht mehr aktuell sind."
+                                    "Seit Version 2.9 unterstützt die Corona-Warn-App auch den Import von Zertifikaten, die eine <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a> dokumentieren. (siehe Blog Artikel: <a href='../../blog/2021-09-08-cwa-version-2-9/' target='_blank' >Corona-Warn-App Version 2.9: Veranstalter*innen können Gäste im Auftrag des Gesundheitsamtes über die Check-in-Funktion warnen</a>)",
+                                    "Seit Version 2.10 erhalten Anwender einen Hinweis, ggf. eine <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a> in Betracht zu ziehen. Der Hinweis basiert auf nationalen Regeln, die in Absprache mit dem Robert Koch-Institut (RKI) implementiert wurden. Diese Regeln werden fortlaufend geprüft und entsprechend den Entwicklungen und neuen Erkenntnissen angepasst. (siehe Blog Artikel: <a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank' >Mit Version 2.10 ist die Corona-Warn-App bereit, schnell über <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfungen</a> zu informieren.</a>)",

```

Comment:
Another illegal nested link here.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/8730e3fa06233ad08580e93253c6a7e9f2f6fb6a, thanks!